### ISSUE TITLE
Not respecting outputPaths config in Brocfile.js

### ISSUE THREAD

**Issue by lukesargeant** [gdub22/ember-cli-less#6]
i'm using ember-cli 0.1.5,

I'd like to build a css file for each 'skin-*.less' file in the app/styles folder. This works with plain CSS using the following config in my Brocfile.js:

```
var app = new EmberApp({
    outputPaths: {
        app: {
            css: {
                'skin-light': '/assets/skin-light.css',
                'skin-dark': '/assets/skin-dark.css'
            }
        }
    }
});
```

After installing the LESS precompiler with 'ember install:addon ember-cli-less' this config is not respected and instead 'ember build' looks for app/styles/app.less.

**Comment by gdub22** [comment]
@lukesargeant yes, the addon doesn't support this yet. Not sure when I can get to adding it, but happy to review a PR if you would like to submit one.

**Comment by lukesargeant** [comment]
@gdub22 gladly, pull request: https://github.com/gdub22/ember-cli-less/pull/7

**Comment by gdub22** [comment]
Closed via #7
